bpf/bpf_lxc.c

```diff
@@ -305,6 +305,7 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 		return ipv6_local_delivery(skb, l3_off, l4_off, SECLABEL, ip6, tuple->nexthdr);
 	} else {
 #ifdef LXC_NAT46
+	  	/* XXX: Is all NATted egress traffic allowed? */
```

Comment:
Looking into this deeper, it seem this case is already covered, as follows:
1. NATted egress traffic tail call into tail_ipv6_to_ipv4()
2. tail_ipv6_to_ipv4() tail calls into tail_handle_ipv4(), which calls handle_ipv4()
3. handle_ipv4() already checks for the egress CIDR policies if outgoing traffic is not otherwise allowed.

Based on this I removed the comment.